
public class ExceptionDemo {

	public static void main(String[] args) {
 
		try {
 
			badMethod();
			 System.out.println("A");
			
		}
		catch (Exception e) {
			System.err.println("B");
		}
		finally {
			System.out.println("C");
		}
		System.out.println("D");
 	}

 public  static void badMethod() { System.out.println("In bad method");}
 }
